### PR TITLE
Release 1.8.0-RC1 back into develop

### PR DESCRIPTION
**Pull request by MarcusBaitz**

### CODE REVIEW

File: frontend/lib/js/form-components/ClearableInput.js [hunk 1]
```diff
@@ -2,28 +2,53 @@
 
 import React                from 'react';
 import T                    from 'i18n-react';
+import NumberFormat         from 'react-number-format';
 
 import { isEmpty }          from '../common/helpers';
+import { MONEY_RANGE }      from '.';
```
Comment (rs22) [inline frontend/lib/js/form-components/ClearableInput.js]: Maybe a question of taste -- `import {} from '.'` is not very descriptive, we could also import this directly from `'./filterTypes'`

File: frontend/lib/js/form-components/InputRange.js [hunk 2]
```diff
@@ -70,6 +84,11 @@ const InputRange = (props: PropsType) => {
           min: value ? value.min : null,
           max: value ? value.max : null,
           [type]: nextValue
+        },
+        {
+          min: props.input.formattedValue ? props.input.formattedValue.min : null,
+          max: props.input.formattedValue ? props.input.formattedValue.max : null,
+          [type]: formattedValue
```
Comment (rs22) [inline frontend/lib/js/form-components/InputRange.js]: I don't know why -- but when I was testing, the x-Buttons inside the input elements didn't have the expected effect. 

When resetting in exact mode, nothing happens, when in range-mode, only the formattedValue is reset but not the underlying raw value.

File: frontend/app/api/concepts.json [hunk 3]
```diff
@@ -59,10 +59,20 @@
             {
               "id": "budget",
               "label": "Budget",
-              "type": "INTEGER_RANGE",
+              "type": "MONEY_RANGE",
               "unit": "1K USD",
```
Comment (rs22) [inline frontend/app/api/concepts.json]: This doesn't make sense when we're now entering Euro values.

File: frontend/lib/js/form-components/filterTypes.js [hunk 4]
```diff
@@ -14,6 +15,7 @@ export const SUPPORTED_FILTERS = {
   REAL_RANGE: REAL_RANGE,
   STRING: STRING,
   BIG_MULTI_SELECT: BIG_MULTI_SELECT,
+  MONEY_RANGE: MONEY_RANGE,
   // Maybe later:
   // 'MONEY_RANGE'
```
Comment (rs22) [inline frontend/lib/js/form-components/filterTypes.js]: Nitpick: Could you remove this comment line, now that we have a `MONEY_RANGE`? Could you also add `MONEY_RANGE` below `REAL_RANGE` above, so that all range filter types are grouped together?

File: frontend/lib/js/query-node-editor/ParameterTableFilters.js [hunk 5]
```diff
@@ -120,7 +121,6 @@ const ParameterTableFilters = (props: PropsType) => (
                     inputType="number"
                     input={{
                       value: filter.value,
-                      defaultValue: filter.defaultValue,
```
Comment (rs22) [inline frontend/lib/js/query-node-editor/ParameterTableFilters.js]: Why did you remove this?

File: frontend/app/api/concepts.json (hunk 3 above)
Comment (MarcusBaitz) [reply frontend/app/api/concepts.json]: This will render the correct filter type.

File: frontend/lib/js/query-node-editor/ParameterTableFilters.js (hunk 5 above)
Comment (MarcusBaitz) [reply frontend/lib/js/query-node-editor/ParameterTableFilters.js]: I ask myself the same thing. I did not consciously remove that.

File: frontend/lib/js/form-components/InputRange.js (hunk 2 above)
Comment (MarcusBaitz) [reply frontend/lib/js/form-components/InputRange.js]: Should be correct now

**Comment by MarcusBaitz** [comment]
I had a branch on money range open, but by making a new delivery tomorrow, I've already merged the feature branch into the release branch.

File: frontend/app/api/concepts.json (hunk 3 above)
Comment (rs22) [reply frontend/app/api/concepts.json]: I meant the `"unit": "1K USD"` line, I'll fix this online

File: frontend/.eslintrc.json [hunk 6]
```diff
@@ -24,7 +24,7 @@
     "comma-dangle": 0, // May be there, helps better diffs and easier maintainability
     "indent": 0,                 // We indent how we like. (sometimes a custom indentation is great)
     "jsx-quotes": [2, "prefer-double"],
-    "max-len": [2, 100],
+    "max-len": [2, 120],
```
Comment (rs22) [inline frontend/.eslintrc.json]: Was there a reason to change this?

File: frontend/app/api/concepts.json [hunk 7]
```diff
@@ -59,10 +59,15 @@
...
               "description": "The movie's total budget",
-              "min": 1
+              "min": 1,
+              "defaultValue": {
```
Comment (rs22) [inline frontend/app/api/concepts.json]: I still think we should rather demo this in the example stored query, instead of always settings this default value. 
What movie has a budget smaller than 44,55 Euros? :wink:
Also, the value for `exact` has to be removed.
